SQLTools.sublime-settings

```diff
@@ -15,17 +17,38 @@
         "sqlite"  : "sqlite3",
         "vertica" : "vsql",
         "firebird": "isql",
-        "sqsh"    : "sqsh"
+        "sqsh"    : "sqsh",
```

Comment:
The trailing commas are not a valid JSON

Reply:
This file is not JSON file, it is a `.sublime-settings` file. And trailing commas are valid. If this is supposed to be a valid JSON, it should be named `.json` instead of `.sublime-settings`.

Trailing commas are a good practice, because you can reorder itens easily. Most time you reorder something, you will have to manually fix the trailing comma.

Reply:
Sublime Text does not use trailing commas in its own config files and neither will do SQLTools.
Sublime Text does use the comments in its own config files and SQLTools will do so as well.

Reply:
Sublime Text team does not follow some good practices, you refuses to follow to? Ok.